File: ai/examples/runtime_error.py
def solution(a, b):
    return a / b

print(solution(10, 0))

{
  "problem_title": "나누기 함수",
  "problem_description": "두 수를 나누는 함수",
  "language": "python",
  "student_code": "def solution(a, b):\n    return a / b\n\nprint(solution(10, 0))",
  "test_result": "Runtime Error",
  "failed_cases": [
    {
      "input": "10 0",
      "expected_output": "정상 출력",
      "actual_output": "ZeroDivisionError",
      "reason": "0으로 나눔"
    }
  ],
  "judge_message": "런타임 에러 발생"
}

File: ai/examples/runtime_error.java
{
  "problem_title": "나누기",
  "problem_description": "두 수를 나누는 프로그램",
  "language": "java",
  "student_code": "public class Main {\n    public static void main(String[] args) {\n        int a = 10;\n        int b = 0;\n        System.out.println(a / b);\n    }\n}",
  "test_result": "Runtime Error",
  "failed_cases": [
    {
      "input": "",
      "expected_output": "정상 출력",
      "actual_output": "ArithmeticException",
      "reason": "0으로 나눔"
    }
  ],
  "judge_message": "런타임 에러 발생"
}